lib/session.js

```diff
@@ -63,8 +63,16 @@ Session.prototype.enumerateModules = function () {
   return this[modulesPromise];
 };
 
-Session.prototype.enumerateRanges = function (protection) {
-  return this[request]('process:enumerate-ranges', { protection: protection })
+Session.prototype.enumerateRanges = function (protection, options) {
+  options = options || {};
+  scope = options.scope || null;
```

Comment:
You want `var scope` here btw, to avoid writing to a global variable (sorry, my comment had this error initially :) ).